src/main/java/net/javadiscord/javabot/systems/moderation/ModerationService.java

```diff
@@ -380,6 +406,18 @@ private MessageEmbed buildBanEmbed(Member member, String reason, Member bannedBy
 				.build();
 	}
 
+	private MessageEmbed buildBanEmbed(String snowflakeID, String reason, Member bannedBy) {
+		return new EmbedBuilder()
+				.setColor(Color.RED)
+				.setTitle(String.format("%s | Ban", snowflakeID))
+				.addField("User", snowflakeID, true)
+				.addField("Banned by", bannedBy.getAsMention(), true)
+				.addField("Ban Reason", String.format(reasonFormat, reason), false)
+				.setTimestamp(Instant.now())
+				.setFooter(bannedBy.getUser().getAsTag(), bannedBy.getEffectiveAvatarUrl())
+				.build();
+	}
```

Comment:
Same as above. Rename `snowFlakeID` and make it a long instead. 
+ Instead of `Color.RED` use the ErrorColor from the SlashCommandConfig (GuildConfig#getSlashCommand)

Reply:
That is how the oringinal command was and its not for an error so why use the error colour?

Reply:
The reason it is a String is beacuse A:
- Discord IDs do not specify that they are long numbers.
- Discord will accept any "number" in the user portion `i.e 0000034243234`
- Ban does not have an option for long, it does for string. 
![image](https://user-images.githubusercontent.com/47298501/158031111-5c63b9f1-0dc3-4c8b-9cb8-62fcade67683.png)

Reply:
Discord IDs are by default always long numbers. 
JDA kinda defaults it to Strings tho, as `getId` returns a String and `getIdLong` a Long.
Other libraries like javacord do it differently and use longs as the default id, idk.
However, we're using long numbers as ids everywhere so why break the code style and use string instead if it is not necessary.  And I'm pretty sure you can use longs in the ban method. And if not this may be a worthy PR for JDA, cuz they allow basically everywhere to use either strings or longs as user ids.

Reply:
And the Error Color is used in many different places. It's just a red-ish color. It's not exclusivly meant for errors

Reply:
as stated in an earlier comment, "Ban does not have an option for long, it does for string." Cconverting that to whatever, so it can be used as long in other calls just to turn it back would be meaningless.